CONTRIBUTING.md

```diff
@@ -87,6 +87,15 @@ before it will be merged. The basic squashing workflow is shown below.
     # save and quit
     git push -f # (force push to GitHub)
 
+If you need help with squashing (or other workflows with `git`), you can
+alternatively enable "Allow edits from maintainers" in the right GitHub
+sidebar and ask for help in the pull request.
```

Comment:
Well, not in general, but we want to encourage it for people that don't know how to use git. I'd prefer people to learn how to do it themselves, of course, but personally tutoring everyone how to use git (especially those that use the web interface :( ) is a lot of work.